src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+export function sortStudents(students: Student[], sortBy: SortType, order: SortOrder) {
+  const compareFunction = (firstStudent: Student, secondStudent: Student) => {
+    let result = 0;
+
+    switch (sortBy) {
+      case SortType.Name:
+        result = firstStudent.name.localeCompare(secondStudent.name);
+        break;
+      case SortType.Surname:
+        result = firstStudent.surname.localeCompare(secondStudent.surname);
+        break;
```

Comment:
you can join it 
firstStudent[sortType].localeCompare(secondStudent[sortType])

Reply:
not fixed; try to do like 


example:
```suggestion
      case SortType.Name:
      case SortType.Surname:
        result = firstStudent[sortBy].localeCompare(secondStudent[sortBy]);
        break;
```